### ISSUE TITLE
[ErrorException] "continue" targeting switch is equivalent to "break". Did you mean to use "continue 2"?

### ISSUE THREAD

**Issue by ervee** [robertvandenbaar/LDG#10]
Hi!

I'm trying to update my LDG to the latest master (I notice the latest _release_ is not in line with _master_...) but when executing ```sudo -u www-data composer install``` in the app folder I get this error:

```
Loading composer repositories with package information
Installing dependencies (including require-dev) from lock file

  [ErrorException]
  "continue" targeting switch is equivalent to "break". Did you mean to use "continue 2"?

install [--prefer-source] [--prefer-dist] [--dry-run] [--dev] [--no-dev] [--no-custom-installers] [--no-autoloader] [--no-scripts] [--no-progress] [--no-suggest] [-v|vv|vvv|--verbose] [-o|--optimize-autoloader] [-a|--classmap-authoritative] [--ignore-platform-reqs] [--] [<packages>]...
```

I also unpacked into a fresh new folder and tried again but with the same error.

I have ```Debian 9 (stretch)``` with ```PHP 7.3.19``` and ```Composer version 1.2.2 2016-11-03 17:43:15```

PS: We need to do a beer with perhaps some shoot'n pool in the near future :)

**Comment by robertvandenbaar** [comment]
Your composer version looks a little outdated.. You could try `composer self-update` or manually install a newer version of composer. Let me know if this fixes the issue.

And I concur, we should definitely do that :)

**Comment by ervee** [comment]
Okay, updated composer by running "curl -sS https://getcomposer.org/installer | php" as normal user. Then replaced the system installed composer with sudo.

This is now the new output:

```root@xxxxxx:/xxxxxxxxxxx/htdocs/xxxxxgal/app# sudo -u www-data composer install
Deprecation warning: Your package name LDG is invalid, it should have a vendor name, a forward slash, and a package name. The vendor and package name can be words separated by -, . or _. The complete name should match "^[a-z0-9]([_.-]?[a-z0-9]+)*/[a-z0-9](([_.]?|-{0,2})[a-z0-9]+)*$". Make sure you fix this as Composer 2.0 will error.
Loading composer repositories with package information
Installing dependencies (including require-dev) from lock file
Package operations: 3 installs, 0 updates, 0 removals
  - Installing dfridrich/php-mime-type (v2.0.3): Downloading (100%)         
  - Installing twig/twig (v1.35.3): Downloading (100%)         
  - Installing vakata/image (2.0.0): Downloading (100%)         
dfridrich/php-mime-type suggests installing symfony/http-foundation (Allows you to get prepared Symfony response.)
Generating autoload files
```
After this, LDG does work!
